src/python/WMCore/WMSpec/Steps/Executors/CMSSW.py

```diff
@@ -273,7 +275,27 @@ def execute(self, emulator=None):
 
         os.environ.update(envOverride)
 
-        returnCode = subprocess.call(args, stdout=stdoutHandle, stderr=stderrHandle)
+        startTime = time.time()
```

Comment:
It would be great to move all of this under a dedicated method, but given that we have to pass in many parameters and retrieve many out of the method, it might be uglier than keeping this code in this method. Anyhow, if you see I way to isolate all these juggling, please go ahead!

Reply:
Alan, I made code better and safer (by using `with` instead of manual close which can never happen due to try/except block) but I think I didn't find way to make it as independent function though and probably better to keep this way.